### PR TITLE
TLMBLGIS-188 remove delete method from api

### PR DESCRIPTION
**Pull request by dovek**
Due: 9/28
@dlhaines

### CODE REVIEW

**Comment by dlhaines** [comment]
Looks ok on the surface, but since I don't know django or python I can't really tell.  It should be passed by someone with more specific knowledge.

File: mbof/views.py [hunk 1]
```diff
@@ -28,9 +29,24 @@ def log_action(request):
...
     message = '%s : %s %s' % (me, method, url)
     logger.info(message)
-    
 
-class UserViewSet(viewsets.ModelViewSet):
+
+class OcellusViewSet(mixins.CreateModelMixin,
+                       mixins.RetrieveModelMixin,
+                       mixins.UpdateModelMixin,
+                       # mixins.DestroyModelMixin,
```
Comment (lsloan) [inline mbof/views.py]: Remove this commented out class reference.  Add some text that explains why you created the `OcellusViewSet` class.  (That you wanted to leave out the delete capabilities provided by `mixins.DestroyModelMixin`.)

File: mbof/views.py [hunk 2]
```diff
@@ -28,9 +29,24 @@ def log_action(request):
...
+                       mixins.ListModelMixin,
+                       viewsets.GenericViewSet):
+
+
+    """
+    A viewset that provides default `create()`, `retrieve()`, `update()`,
+    `partial_update()`, `destroy()` and `list()` actions.
```
Comment (lsloan) [inline mbof/views.py]: Does it **_really_** provide the `destroy()` method?  Misleading comments are worse than no comment at all.

**Review by lsloan — COMMENTED** [review]
Update the documentation and comments, then it's ready to go.

**Pull request merged**